src/catkin_pkg/cli/generate_changelog.py

```diff
@@ -40,23 +40,32 @@ def prompt_continue(msg, default):
 
 def main(sysargs=None):
     parser = argparse.ArgumentParser(description='Generate a REP-0132 %s' % CHANGELOG_FILENAME)
+    ARG_ONLYMERGE = '--only-merges'
+    ARG_SKIPMERGE = '--skip-merges'
     parser.add_argument(
         '-a', '--all', action='store_true', default=False,
         help='Generate changelog for all versions instead of only the forthcoming one (only supported when no changelog file exists yet)')
+    parser.add_argument(
+        ARG_ONLYMERGE, action='store_true', default=False,
+        help='Skip all commits but merge commits to the changelog')
     parser.add_argument(
         '--print-root', action='store_true', default=False,
         help='Output changelog content to the console as if there would be only one package in the root of the repository')
     parser.add_argument(
         '--skip-contributors', action='store_true', default=False,
         help='Skip adding the list of contributors to the changelog')
     parser.add_argument(
-        '--skip-merges', action='store_true', default=False,
+        ARG_SKIPMERGE, action='store_true', default=False,
         help='Skip adding merge commits to the changelog')
     parser.add_argument(
         '-y', '--non-interactive', action='store_true', default=False,
         help="Run without user interaction, confirming all questions with 'yes'")
     args = parser.parse_args(sysargs)
 
+    # Error when mutually exclusive args are defined.
+    if args.only-merges and args.skip-merges:
```

Comment:
> Added a change, and tested:

CI is failing on these lines so I doubt you ran the latest code locally.

Reply:
Sorry, `pyc`s weren't updated. Also symlink in `PYTHONPATH` didn't seem to help updating the change in modules.

Anyways, sorted those out, tested again. Looks ok to me.

```
$ catkin_generate_changelog --only-merges --skip-merges
usage: catkin_generate_changelog [-h] [-a] [--only-merges] [--print-root]
                                 [--skip-contributors] [--skip-merges] [-y]
catkin_generate_changelog: error: argument --skip-merges: not allowed with argument --only-merges
```
```
$ catkin_generate_changelog --only-merges 
Found packages: pkg_1, pkg_1_support
Querying commit information since latest tag...
Updating forthcoming section of changelog files...
- updating 'pkg_1/CHANGELOG.rst'
- updating 'pkg_1_support/CHANGELOG.rst'
Done.
Please review the extracted commit messages and consolidate the changelog entries before committing the files!
```
```
$ catkin_generate_changelog  --skip-merges
Found packages: pkg_1, pkg_1_support
Querying commit information since latest tag...
Updating forthcoming section of changelog files...
- updating 'pkg_1/CHANGELOG.rst'
- updating 'pkg_1_support/CHANGELOG.rst'
Done.
Please review the extracted commit messages and consolidate the changelog entries before committing the files!
```